index.php

```diff
@@ -1,37 +1,108 @@
+            <?php include 'pdo.php'?>
 
-    </header>
-    <div class="main">
-      <h2>list all events for a particular day</h2>
-      <h2>switch a session's day/time and/or location</h2>
-      <!-- placeholder -->
-      <?php
+            <div id="listDayEvents">
+                <h2>Event Date:</h2>
+                <select id='dateselect' onChange='displayEvent(this.value)'>
+                    <option value=''></option>
+                    <option value='saturday'>saturday</option>
+                    <option value='sunday'>sunday</option>
+                </select><br>
+                <div id="displayevent">
+                <table>
+                    <tr>
+                        <th>Session</th>
+                        <th>Start Time</th>
+                        <th>End Time</th>
+                        <th>Room</th>
+                    <tr>
+                </table>
+            </div>
 
-      ?>
-    </div>
-    <footer>
-      
-    </footer>
-  </body>
+            </div>
+            <div id="eventManipulation">
+                <h2>Select Session:</h2>
+                <div id="sessionselectdiv">
+                <?php
+                    //Gets the possible rooms to select from";
+                    $query = "SELECT DISTINCT sessionName, TIME_FORMAT(startTime,'%h:%i%p') as startTimeReadable, startTime, room FROM SessionEvent;";
+                    $stmt = $pdo->prepare($query);
+                    $stmt->execute();
+                    //Displays a select menu with all unique session events
+                    echo "<select id='sessionselect' name='session' onChange=provideChoices(this.value) form='setEventForm'>";
+                        echo "<option value=''></option>";
+                        while ($session = $stmt->fetch()){
+                            $sessionName = $session["sessionName"];
+                            $startTime = $session["startTime"];
+                            $startTimeReadable = $session["startTimeReadable"];
+                            $room = $session["room"];
+                            echo "<option value='startTime=" . $startTime . "&room=" . $room . "&name=" . $sessionName . "'>" . $sessionName . ", Room: " . $room . ", " . $startTimeReadable . "</option>";
+                        }
+                    echo "</select><br>";
+
+                    ?>
+                </div>
+                <div id="displayoptions"></div>
+            </div>
+        </div>
+        <footer>
+        
+        </footer>
+    </body>
+  <script>
+        //Function is used to display events associated with a specific day
+        function displayEvent(day){
+        //gets the div to display events for specific date.
+            let content  = document.getElementById("displayevent");
+            fetch("./displayEvent.php?day=" + day)
+            .then((response) => {
+                if (response.status == 200){
+                    return response.text().then((text) => {
```

Comment:
a friendly tip: use promise chaining instead of something that resembles a callback hell
```js
fetch("./displayEvent.php?day=" + day)
  .then(respone => {
    if (response.status== 200)
      return response.text();
  })
  .then(text => {
    content.innerHTML = text;
  })
  .catch(errorFn);
```

Reply:
Oooh I like this, was confused before as to how chaining works but this makes a lot of sense. Nice and clean.